modules/genomic_browser/php/models/filesdto.class.inc

```diff
@@ -0,0 +1,50 @@
+<?php declare(strict_types=1);
+/**
+ * PHP version 7
+ *
+ * @category Genomic_Browser
+ * @package  Main
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ *           Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+namespace LORIS\genomic_browser\Models;
+
+/**
+ * A FilesDTO represents an Files in the genomic browser module.
+ * It is doing the mapping between the profile table columns
+ * and the JSON object properties returned to the frontend.
+ *
+ * Additionally, it implements the DataInstance interface so it can be used by
+ * a Database Provisioner.
+ *
+ * @category Genomic_Browser
+ * @package  Main
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ *           Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+class FilesDTO implements \LORIS\Data\DataInstance
+{
+    /**
+     * Implements \LORIS\Data\DataInstance interface for this row.
+     *
+     * @return array
+     */
+    public function jsonSerialize() : array
+    {
+        return [
+            'GenomicFileID'    => $this->GenomicFileID,
```

Comment:
I don't see how any of these class variables are ever set. There's no constructor.. (same with all the other ones in this directory..)

The class variables used/needed should also be defined explicitly..